### ISSUE TITLE
Remove empty destructors where needed

### ISSUE THREAD

**Issue by neurocod** [horde3d/Horde3D#129]
All QObject-derived classes have virtual destructors, so there is no need for empty destructor stubs in many files, like
```CameraToolBar::~CameraToolBar()
{
}```

**Comment by algts** [comment]
Hello. This issue is more of personal preference, I think. Many programmers like to clearly define constructors/destructors, so you always know what is the behaviour of the specific class.
It is probably low priority for now, but you can always pull request your changes.

**Comment by neurocod** [comment]
Yeah, I know it's low priority, just wanted to know opinion before possible PR. I don't agree with another part - after defining empty destructor, one needs to look at it's definition to ensure that it makes nothing, just like absent destructor. Without explicit destructor programmer knows default deletion behaviour, with declaration programmer becomes unaware with what is going on untill peeks definition.
